File: cases/taint/python/return-relay-one-hop-negative/return_relay.py
def dfb_source():  # DFB-SOURCE: return-one-hop-negative-input
    return "tainted"


def relay(value):  # DFB-WITNESS: return-one-hop-negative-relay
    return value


def dfb_sink(value):  # DFB-SINK: return-one-hop-negative-sink
    pass


def run():
    result = relay(dfb_source())
    dfb_sink("clean")

File: cases/taint/ruby/return-relay-one-hop-negative/return_relay.rb
def dfb_source # DFB-SOURCE: return-one-hop-negative-input
  "tainted"
end

def relay(value) # DFB-WITNESS: return-one-hop-negative-relay
  value
end

def dfb_sink(value) # DFB-SINK: return-one-hop-negative-sink
end

def run
  result = relay(dfb_source)
  dfb_sink("clean")
end
